internal/backends/postgresql/metadata/registry_test.go

```diff
@@ -150,7 +150,9 @@ func TestCreateDrop(t *testing.T) {
 		t.Skip("skipping in -short mode")
 	}
 
-	t.Parallel()
+	// Do not run parallel, `initDBs()` searches all schema with ferretdb metadata table and
+	// including schema created by other tests that can be deleted before `initCollections()`
+	// is called would cause pg relation does not exist error.
```

Comment:
See https://github.com/FerretDB/FerretDB/actions/runs/6309012677/job/17128119042?pr=3419

Reply:
This will fix second case of https://github.com/FerretDB/FerretDB/issues/3409

Reply:
will be done in https://github.com/FerretDB/FerretDB/pull/3438